core/arch/arm/plat-rockchip/platform_config.h

```diff
@@ -85,6 +85,25 @@
 #define FIREWALL_DDR_BASE	0xff534000
 #define FIREWALL_DDR_SIZE	SIZE_K(16)
 
+#elif defined(PLATFORM_FLAVOR_rk3588)
+
+#define GIC_BASE		0xFE600000
+#define GIC_SIZE		SIZE_K(64)
+#define GICD_BASE		GIC_BASE
+#define GICC_BASE		0
+
+#define UART1_BASE		0xfeb40000
+#define UART1_SIZE		SIZE_K(64)
+
+#define UART2_BASE		0xfeb50000
+#define UART2_SIZE		SIZE_K(64)
+
+#define UART5_BASE		0xfeb80000
+#define UART5_SIZE		SIZE_K(64)
+
+#define FIREWALL_DDR_BASE	0xfe030000
```

Comment:
I see rk3588 has two firewalls: `FIREWALL_DDR` with base address `0xFE030000` and `FIREWALL_SYSMEM` with base address `0xFE038000` have you been able to find out the difference? I guess the `FIREWALL_DDR` hold MMU registers, and `FIREWALL_SYSMEM` holds SMMU registers, but I cannot confirm this.